package.json

```diff
@@ -18,16 +18,16 @@
   },
   "dependencies": {
     "@lumino/algorithm": "^1.3.3",
-    "@lumino/datagrid": "^0.12.0",
-    "@lumino/default-theme": "^0.4.4",
-    "@lumino/widgets": "^1.13.4",
+    "@lumino/datagrid": "^0.14.0",
+    "@lumino/default-theme": "^0.5.0",
+    "@lumino/widgets": "^1.14.0",
```

Comment:
Updating the dependencies too since it's a little related to Lumino. Everything appears to be working OK :+1: